interview-questions.md

```diff
@@ -14,44 +14,47 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is a softwre used in your terminal to push/merge/ pull code onto github.
+                github is a gui you use to help manage your branches/ code that you have pushed or pulled
 
 Researched answer:
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: <|| && !> are all operators that return a boolean value (true/false)
 
-Researched answer:
+Researched answer: Google says the same :)
```

Comment:
Haha yes ... BUT if you were already correct, elaborate on it. If you were able to add any more context to your original answer what would it be? (a thought for next assessment)